shared/common-adapters/dumb.desktop.js

```diff
@@ -1,10 +1,21 @@
 // @flow
-import Checkbox from './checkbox'
 import React, {Component} from 'react'
 import _ from 'lodash'
 import type {DumbComponentMap} from '../constants/types/more'
 import type {IconType} from './icon.constants'
-import {Button, Box, TabBar, Text, Avatar, ListItem, PopupMenu, Icon} from './index'
+import {
+  Avatar,
+  Button,
+  Box,
+  Checkbox,
+  ChoiceList,
+  Icon,
+  ListItem,
+  PopupMenu,
+  StandardScreen,
+  TabBar,
+  Text,
+} from './index'
```

Comment:
@cjb I was thinking that the git diffs would be more useful if it was like this, but it conflicts with the style in the rest of the codebase so I'll revert it.